Package.swift

```diff
@@ -1,16 +1,23 @@
-// swift-tools-version:5.0
+// swift-tools-version:5.1
 import PackageDescription
 
 let package = Package(
   name: "PovioKit",
   platforms: [
-    .macOS(.v10_14), .iOS(.v11), .tvOS(.v10), .watchOS(.v3)
+    .iOS(.v11),
+    .macOS(.v10_14),
```

Comment:
Should we disable `macOS` since we are using UIKit in several places?

Reply:
Good point. It was added when we were also using Alamofire dependency with SPM and it required macos platform. I'll remove it.